### PR TITLE
Chore/gpu/pbs throughput bench gpu

### PR DESCRIPTION
**Pull request by agnesLeroy**
closes: _please link all relevant issues_

### Check-list:

* [ ] Tests for the changes have been added (for bug fixes / features)
* [ ] Docs have been added / updated (for bug fixes / features)
* [ ] Relevant issues are marked as resolved/closed, related issues are linked in the description
* [ ] Check for breaking changes (including serialization changes) and add them to commit message following the conventional commit [specification][conventional-breaking]

[conventional-breaking]: https://www.conventionalcommits.org/en/v1.0.0/#commit-message-with-description-and-breaking-change-footer

### CODE REVIEW

**Comment by agnesLeroy** [comment]
@slab-ci bench_pbs_gpu

**Comment by agnesLeroy** [comment]
@slab-ci bench_core_crypto_gpu

**Comment by agnesLeroy** [comment]
@slab-ci core_crypto_gpu_bench

**Comment by agnesLeroy** [comment]
@slab-ci core_crypto_gpu_bench

**Comment by agnesLeroy** [comment]
@slab-ci core_crypto_gpu_bench

**Comment by agnesLeroy** [comment]
@slab-ci bench_core_crypto_gpu

File: tfhe/benches/core_crypto/pbs_bench.rs [hunk 1]
```diff
@@ -850,6 +850,287 @@ mod cuda {
...
+                    black_box(&mut out_pbs_ct_gpu);
+                })
+            });
+
+            let bit_size = params.message_modulus.unwrap().ilog2();
+            write_to_json(
+                &id,
+                *params,
+                name,
+                "pbs_throughput",
```
Comment (soonum) [inline tfhe/benches/core_crypto/pbs_bench.rs:984]: Just `"pbs"` the throughput notion is a benchmark detail. But the operation is ultimately just a PBS.

File: tfhe/benches/core_crypto/pbs_bench.rs [hunk 2]
```diff
@@ -850,6 +850,287 @@ mod cuda {
...
```
Comment (soonum) [inline tfhe/benches/core_crypto/pbs_bench.rs:1126]: Same here `"pbs"`

**Review by soonum — COMMENTED** [review]
Nice you've added the multi-bit throughput for the GPU. I suppose I'll have to add the same benchmark for the CPU.

**Comment by agnesLeroy** [comment]
@slab-ci core_crypto_gpu_bench

**Comment by agnesLeroy** [comment]
@slab-ci core_crypto_bench

**Comment by agnesLeroy** [comment]
@slab-ci core_crypto_bench_gpu

**Comment by agnesLeroy** [comment]
@slab-ci core_crypto_gpu_bench

**Comment by agnesLeroy** [comment]
@slab-ci core_crypto_gpu_bench

**Pull request merged**